consensus_pledge_model/logic.py

```diff
@@ -289,37 +289,37 @@ def s_sectors_rewards(_1,
     & reward_schedule_init = {0: 20, 1: 30, 2: 40, 3: 50}
     then
     reward_schedule_final = {0: 20 + 5, 1: 30 + 5, 2: 40 + 5, 3: 50}
-    
+
     """
-    #retrieve total rewards
-    total_reward = Reward("block_reward")
-    linear_duration = ConsensusPledgeParams("linear_duration")
+    # retrieve total rewards
+    total_reward = state["reward"]
```

Comment:
state["reward"] is of `Reward` type. The actual `float` Block Reward is `state["reward"].block_reward` which is want you would want to use here.

https://github.com/BlockScience/filecoin-consensus-pledge-demo/blob/45b512c2ceb9f8c62d836dc7e68e416b869e0950/consensus_pledge_model/types.py#L56